modules/search-service-impl/src/main/java/org/opencastproject/search/impl/solr/SolrRequester.java

```diff
@@ -209,6 +213,18 @@ public MediaPackage getMediaPackage() {
           return null;
         }
 
+        @Override
+        public AccessControlList getAccessControlList() {
+          final List<AccessControlEntry> entries = Schema.getOcAcl(doc)
+              .stream()
+              .flatMap(field -> {
+                return Arrays.stream(field.getValue().strip().split("\\s+"))
+                    .map(role -> new AccessControlEntry(role, field.getSuffix(), true));
+              })
+              .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
Just out of curiosity. Any reason to specifically choose `ArrayList` here?
I would just have used `.collect(Collectors.toList())`.